src/parse_args.ts

```diff
@@ -38,10 +38,15 @@ program
     'configuration path (default: synthetics.config.js)'
   )
   .option(
-    '-p, --params <jsonstring>',
+    '-p, --params <jsonstring>, --suite-params <jsonstring>',
```

Comment:
I am confused with this notation, What does this do?

Reply:
Oops, let me remove it. It does nothing. You can't add a third form of the command, I needed to add a new option below.